examples/upgrade/blue-green-route53/eks-green/main.tf

```diff
@@ -0,0 +1,55 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+provider "kubernetes" {
+  host                   = module.eks_cluster.eks_cluster_endpoint
+  cluster_ca_certificate = base64decode(module.eks_cluster.eks_cluster_certificate_authority_data)
+  token                  = data.aws_eks_cluster_auth.this.token
+}
+
+provider "helm" {
+  kubernetes {
+    host                   = module.eks_cluster.eks_cluster_endpoint
+    cluster_ca_certificate = base64decode(module.eks_cluster.eks_cluster_certificate_authority_data)
+    token                  = data.aws_eks_cluster_auth.this.token
+  }
+}
+
+provider "kubectl" {
+  apply_retry_count      = 10
+  host                   = module.eks_cluster.eks_cluster_endpoint
+  cluster_ca_certificate = base64decode(module.eks_cluster.eks_cluster_certificate_authority_data)
+  load_config_file       = false
+  token                  = data.aws_eks_cluster_auth.this.token
+}
+data "aws_eks_cluster_auth" "this" {
+  name = module.eks_cluster.eks_cluster_id
+}
+
+module "eks_cluster" {
+  source = "../modules/eks_cluster"
+
+  suffix_stack_name = "green"
+  cluster_version   = "1.24"
```

Comment:
It would be nice to add a comment here to highlight the bump in the cluster version.

Reply:
added